submissions/chabuChabu/html/contact-form.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" >
+        <title>Contact-Us</title>
+    </head>
+    <body >
+        <h1>Kwathu</h1><hr
+        <div>
+          <a href="signup.html">Sign up</a>
+          <a href="product.html">Product</a>  
+        </div>
+        <h4>Thank you for getting in touch. Please fill out your details and type your inquiry in the form below, A member of our team will get in touch.</h4>
+        <form action="#" method="post" enctype="multipart/form-data">
```

Comment:
You need to remove the attributes `action`, `method` and `enctype` from the form tag.